numpyro/hmc_util.py

```diff
@@ -561,16 +561,35 @@ def _potential_energy(params):
     return _potential_energy
 
 
-def transform_fn(transforms, params, invert=False):
-    return {k: transforms[k](v) if not invert else transforms[k].inv(v)
+def transform_fn(inv_transforms, params, constrain=True):
+    return {k: inv_transforms[k](v) if constrain else inv_transforms[k].inv(v)
             for k, v in params.items()}
 
 
 def initialize_model(rng, model, *model_args, **model_kwargs):
+    """
+    Given a model with Pyro primitives, returns a function which, given
+    unconstrained parameters, evaluates the potential energy (negative
+    joint density). In addition, this also returns initial parameters
+    sampled from the prior to initiate MCMC sampling and functions to
+    transform unconstrained values at sample sites to constrained values
+    within their respective support.
+
+    :param jax.random.PRNGKey rng: random number generator seed to
+        sample from the prior.
+    :param model: Python callable containing Pyro primitives.
+    :param `*model_args`: args provided to the model.
+    :param `**model_kwargs`: kwargs provided to the model.
+    :return: tuple of (`init_params`, `potential_fn`, `inv_transforms`)
+        `init_params` are values from the prior used to initiate MCMC.
+        `inv_transforms` are inverse transform functions that are useful
```

Comment:
I think that `inv_transforms` is a callable which uses inverse transform functions to convert HMC ...

Reply:
Sorry, my documentation is based on the outdated functionality when we were returning transforms as dicts. Will change.